database/generateData.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable func-names */
+const faker = require('faker');
+const db = require('./index');
+
+// generate fake data for 100 unique businesses
+const generateBusiness = function () {
+  const businesses = [];
+  for (let i = 0; i < 100; i += 1) {
+    const entry = {
+      name: faker.company.companyName(),
+      claimed: Math.floor(Math.random() * Math.floor(2)),
+      // overallRating needs future refactor to generate dynamic rating
+      overallRating: Math.floor(Math.random() * Math.floor(5)) + 1,
+      // totalReviews needs future refactor to generate dynamic count
+      totalReviews: Math.floor(Math.random() * Math.floor(60)) + 10,
+      averageCost: Math.floor(Math.random() * Math.floor(4)) + 1,
+      businessType: `${faker.company.catchPhraseAdjective()}, ${faker.name.jobArea()}`,
+      address1: `${faker.address.streetAddress()}`,
```

Comment:
It seems you don't need to wrap the faker call in the back-tic since `faker.address.streetAddress()` will return a String

Reply:
Good catch. I had done some edits to the structure of my generation function, and in the previous iteration, I needed the back-tics. Just edited out the unnecessary code now.